SPECS/safec.spec

```diff
@@ -0,0 +1,108 @@
+#ciscosafec version we are going to release
```

Comment:
Do we have any devel packages needed or did we decided to make this seperate?  I can't remember but I thought there were headers and mentioned documentation (doxgen generated) we could install.  Only ask, can be addressed in next commit to make this one smaller though.

Reply:
Yeah there is doxygen documentation we can turn on.  But I agre think we could address them after we make sure this is working.